scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -98,9 +99,13 @@ def main():
 
     # Load all metrics files.
     if args.metrics_dir:
+        is_header_compatible_multiple_files(parser, [args.in_mask] +
+                                            list_metrics_files)
         metrics_files = [nib.load(args.metrics_dir + f)
                          for f in sorted(os.listdir(args.metrics_dir))]
```

Comment:
Tested. Your code works, but the line below, not.
`args.metrics_dir + f` should be os.path.join(args.metrics_dir, f)`.

If I use `--metrics_dir my_path` instead of `--metrics_dir my_path/'`, it fails.

Otherwise it works.

Reply:
Nice catch ! This was a bug in the current version.